compiler/crates/relay-config/src/project_config.rs

```diff
@@ -62,6 +62,9 @@ pub struct RemotePersistConfig {
         deserialize_with = "deserialize_semaphore_permits"
     )]
     pub semaphore_permits: Option<usize>,
+
+    #[serde(default)]
+    pub include_query_text: bool,
```

Comment:
Should we support this for `LocalPersistConfig` as well for symmetry?

Reply:
If so, what about adding a method to `PersistConfig`?


```rust
impl PersistConfig {
    pub fn include_query_text(&self) -> bool {
        match self {
            PersistConfig::Remote(remote_config) => remote_config.include_query_text,
            PersistConfig::Local(local_config) => local_config.include_query_text,
        }
    }
}
```

Reply:
@tomgasson  @captbaritone  discovering the feature with the release note, but it's me or local persist isn't supported? 
is there a reason for that?

Reply:
It should be:
```js
persist: {
  file: "foo.json",
  includeQueryText: true
}
```
What issue are you having?